dev/com.ibm.ws.security.acme_fat/fat/src/com/ibm/ws/security/acme/docker/ChalltestsrvContainer.java

```diff
@@ -13,29 +13,19 @@
 
 import static junit.framework.Assert.fail;
 
-import java.io.IOException;
 import java.util.Map.Entry;
 
-import org.apache.http.StatusLine;
-import org.apache.http.client.methods.CloseableHttpResponse;
-import org.apache.http.client.methods.HttpPost;
-import org.apache.http.entity.ContentType;
-import org.apache.http.entity.StringEntity;
-import org.apache.http.impl.client.CloseableHttpClient;
-import org.apache.http.impl.client.HttpClients;
-import org.testcontainers.containers.GenericContainer;
 import org.testcontainers.containers.Network;
 import org.testcontainers.containers.output.OutputFrame;
 
 import com.github.dockerjava.api.model.ContainerNetwork;
 import com.ibm.websphere.simplicity.log.Log;
-import com.ibm.ws.security.acme.utils.AcmeFatUtils;
 
 /**
  * Testcontainer implementation for the letsencrypt/pebble-challtestsrv
  * container.
  */
-public class ChalltestsrvContainer extends GenericContainer<ChalltestsrvContainer> {
+public class ChalltestsrvContainer extends CAContainer {
```

Comment:
I wasn't really expecting both PebbleContainer and CAContainer to extend CAContainer. My expectation was that the There would be a new class that would wrap both the PebbleContainer and ChalltestsrvContainer that would extend the CAContainer. This class would then call through appropriately.